doc/release-notes.md

```diff
@@ -20,361 +22,179 @@ How to Upgrade
 If you are running an older version, shut it down. Wait until it has completely
 shut down (which might take a few minutes for older versions), then run the
 installer (on Windows) or just copy over /Applications/Dash-Qt (on Mac) or
-dashd/dash-qt (on Linux).
+dashd/dash-qt (on Linux). Because of per-UTXO fix (see below) there is a
+one-time database upgrade operation, so expect a slightly longer first run.
```

Comment:
Because of **the** per-UTXO fix (see below) there is a one-time database upgrade operation, so expect a slightly longer **startup time on** first run.